public-ui/src/app/highlight-search/highlight-search.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-highlight-search',
+  templateUrl: './highlight-search.component.html',
+  styleUrls: ['./highlight-search.component.css']
+})
+export class HighlightSearchComponent implements OnInit, OnChanges {
+  @Input() text: string;
+  @Input() searchTerm: string;
+  @Input() isSource: boolean;
+  @Input() conceptId: string;
+  @Input() isStandard: string;
+  @Input() indNum: number;
+  chunks: string[];
```

Comment:
Since you are splitting on words, I probably like `words` better as a var name

Reply:
Done.